src/screencastPanel.ts

```diff
@@ -47,6 +61,29 @@ export class ScreencastPanel {
         this.panel.webview.onDidReceiveMessage(message => {
             this.panelSocket.onMessageFromWebview(message);
         }, this);
+
+        this.recordEnumeratedHistogram('DevTools.ScreencastToggle', 1);
+    }
+
+    private recordEnumeratedHistogram(actionName: string, actionCode: number) {
+        const properties: ITelemetryProps = {};
+        properties[`${actionName}.actionCode`] = actionCode.toString();
+        this.telemetryReporter.sendTelemetryEvent(
+            `devtools/${actionName}`,
+            properties);
+        // eslint-disable-next-line no-console
+        console.log({name: `devtools/${actionName}`, properties});
```

Comment:
Remove this log and the one in `recordPerformanceHistogram`.

Reply:
Done